web/modules/custom/react_tools/tools/.vscode/launch.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
hey, @Jsarnik does this file need to be gitignored? It seems to me that it's for the IDE, but not 100% sure.

Reply:
yes it is for the IDE, sorry It should be gitignored, we should have a global gitignore that ignores all .vscode folders and it's files.

Reply:
yeah, that sounds good, please go ahead and add it. We have something similar for PhpStorm files.

Reply:
okay updated PR